### PR TITLE
Fix #184091 www.cooppank.ee

### LINKED ISSUE #184091: www.cooppank.ee
### Issue URL (Incorrect Blocking)
 [https://www.cooppank.ee/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.cooppank.ee%2F)

### Screenshots
<details>
  <summary>Screenshot 1 </summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 1 with AdGuard disabled</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard Browser Extension v4.3.53
System version: | Windows 11
Browser: | Brave
Stealth mode: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Cookie Notices, <br/>AdGuard Popups, <br/>AdGuard Mobile App Banners, <br/>AdGuard Other Annoyances, <br/>AdGuard Widgets<br/><br/><b>Language-specific:</b><br/>Estonian List

### PR DESCRIPTION
**Pull request by alexx7311**

### CODE REVIEW

**Review by piquark6046 — COMMENTED** [review]
I think that you can remove it using `#$#` syntax.
Of course, you need to keep the current rules for free iOS users and AdGuard content blocker users.

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt [hunk 1]
```diff
@@ -13,6 +13,9 @@
 ||geolocation.onetrust.com/cookieconsentpub/*/geo/location$xmlhttprequest,redirect-rule=noopjson
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/184091
+cooppank.ee#@##cookie-modal
```
Comment (zloyden) [inline AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:17]: The cookies should also be blocked, but like @piquark6046 said. You can try using `#$#` rules to do this.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:17]: @piquark6046 I see the horizontal scrolling at the bottom after adding the css rules and an issue with the video, so `trusted-set-cookie` scriptlet seems to be safer here.

Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt:17]: I'd just add this to specific.txt.
```adblock
cooppank.ee#%#//scriptlet('trusted-set-cookie', 'cookie-consent', 'necessary%2B1721979799357')
```

Comment (zloyden) [reply AnnoyancesFilter/Cookies/sections/cookies_allowlist.txt]: @alexx7311 please add the exceptions back)

**Review by zloyden — APPROVED** [review]

**Pull request merged**